Frends.Community.Azure.Blob/Definition.cs

```diff
@@ -1,48 +1,10 @@
 ﻿#pragma warning disable 1591
 
-using System.ComponentModel;
-using System.ComponentModel.DataAnnotations;
-
 namespace Frends.Community.Azure.Blob
 {
-    /// <summary>
-    /// Parameters class usually requires parameters that are required.
-    /// </summary>
-    public class Parameters
-    {
-        /// <summary>
-        /// Something that will be repeated.
-        /// </summary>
-        [DisplayFormat(DataFormatString = "Text")]
-        [DefaultValue("Lorem ipsum dolor sit amet.")]
-        public string Message;
-    }
-
-    /// <summary>
-    /// Options class provides additional parameters.
-    /// </summary>
-    public class Options
-    {
-        /// <summary>
-        /// Number of times input is echoed.
-        /// </summary>
-        [DefaultValue(3)]
-        public int Amount;
-
-        /// <summary>
-        /// How repeats of input are separated.
-        /// </summary>
-        [DisplayFormat(DataFormatString = "Text")]
-        [DefaultValue(" ")]
-        public string Delimiter;
-    }
-
-    public class Result
+    public enum ConnectionMethod
     {
-        /// <summary>
-        /// Contains input repeated specified times.
-        /// </summary>
-        [DisplayFormat(DataFormatString = "Text")]
-        public string Replication;
+        ConnectionString,
+        AccessToken
```

Comment:
Should this be called "Client Credentials" or "OAuth"? Because user does not provide the token, but instead provides the client credentials / utilizes OAuth for auth.